source/includes/library-reference/_brain.html.md

```diff
@@ -13,73 +13,54 @@ such that ">>" indicates a decreasing order of precedence. Note that failure to
 BRAIN name in at least one of these places will result in a friendly error.
 
 ## Brain(config, name)
-Creates a local object for interacting with an existing BRAIN on the server.
-
-```cpp
-auto config = make_shared<bonsai::Config>(argc, argv);
-auto brain = make_shared<bonsai::Brain>(config);
-std::cout << brain << std::endl;
-```
 
 ```python
 config = bonsai_ai.Config(sys.argv)
 brain = bonsai_ai.Brain(config)
 print(brain)
 ```
 
+Creates a local object for interacting with an existing BRAIN on the server.
+
 | Argument | Description |
 | ---      | ---         |
-| `config` | Object returned by previously created `Bonsai::Config` (C++) or `bonsai_ai.Config()` (Python). |
+| `config` | Object returned by previously created `bonsai_ai.Config()`. |
```

Comment:
remove "()" from bonsai_ai.Config since it refers to the class itself rather than its constructor